<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DEEP-1808222042" ignored="0" name="Test" tests="3" failures="1" timestamp="27 五月 2019 10:39:42 GMT" time="62.69" errors="0">
  <testcase name="testOne" time="8.684" classname="com.mail.test.DownloadTest2"/>
  <testcase name="testOne" time="32.112" classname="com.mail.test.DownloadTest"/>
  <testcase name="uplod" time="20.775" classname="com.mail.test.FileUploadVerify">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.mail.test.FileUploadVerify.uplod(FileUploadVerify.java:99)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:745)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- uplod -->
</testsuite> <!-- Test -->
